airbyte-persistence/job-persistence/src/main/java/io/airbyte/persistence/job/DefaultJobPersistence.java

```diff
@@ -1378,8 +1386,10 @@ private static Object getJsonNodeValue(final JsonNode jsonNode, final String col
       return valueNode.asDouble();
     } else if (nodeType == JsonNodeType.NULL) {
       return null;
+    } else if (nodeType == JsonNodeType.MISSING) {
```

Comment:
Did you run into valid cases where we'd hit nodeType is MISSING? Not sure when that would happen, I am wondering if throwing an error could be safer.

Reply:
The `DefaultJobPersistenceTest#Should have valid yaml schemas in exported database` test failed because the `JsonNodeType.MISSING` was not in that if/else statement, resulting in it falling through to the exception below it.  The test passed before the update, so I added this to maintain the test.  Whether or not the test is valid is a separate discussion.

Reply:
This is testing `jobPersistence.exportDatabase()` which I believe has been deprecated. @davinchia, may be you can help confirm this.
It might be better to delete the whole test rather than be more permissive regarding unexpected Json data.